lib/http/options.rb

```diff
@@ -50,7 +50,8 @@ def initialize(options = {})
         :keep_alive_timeout => 5,
         :headers            => {},
         :cookies            => {},
-        :encoding           => nil
+        :encoding           => nil,
+        :logger             => HTTP::Logger.new
```

Comment:
Yeah but you will be allocating new instance everytime Options got initialized, which is not pretty awesome ;))